BsuirApi/Models/Employee.swift

```diff
@@ -29,7 +29,7 @@ public struct Employee: Codable, Equatable, Identifiable, Hashable {
 
 extension Employee {
     public var fio: String {
-        return [lastName, firstName, middleName]
+        return [lastName, firstName, middleName ?? ""]
             .filter { !$0.isEmpty }
```

Comment:
оператор `??` иногда очень долго компилируется потому что компилятору очень много типов надо проверить, лучше по возможности его избегать
```suggestion
        return [lastName, firstName, middleName]
            .compactMap { name in 
                guard let name = name, !name.isEmpty else { return nil }
                return name
            }
```